gdal/frmts/netcdf/netcdflayersg.cpp

```diff
@@ -74,36 +74,42 @@ namespace nccfdriver
     {
         geom_t ret = NONE;
 
-        if (type == wkbPoint || type == wkbPoint25D)
+        if (type == wkbPoint || type == wkbPoint25D || type == wkbPointM || type == wkbPointZM)
```

Comment:
intead of testing all those combinations you could just have `auto eFlatType = wkbFlatten(type)`, and just test for eFlatType = wkbPoint, wkbLineString, etc.

Reply:
done